### PR TITLE
Add tile display to the UI

### PR DESCRIPTION
**Pull request by jonathan-marmelab**
Capture d’écran du 2023-02-28 10-26-13

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,6 +47,9 @@ test: 									## Run unit tests
 	@(${MAKE} -C domain test)
 	@(${MAKE} -C webapp test)
 
+test-webapp:							## Run unit tests for the webapp
+	@(cd webapp && composer install && bin/phpunit)
```
Comment (slax57) [inline Makefile:51]: Why not call `@(${MAKE} -C webapp test)` here? :thinking:

File: webapp/config/packages/monolog.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 monolog:
     channels:
         - deprecation # Deprecations are logged in the dedicated "deprecation" channel when it exists
+    handlers:
+        deprecation:
+            type: "null"
+            channels: [deprecation]
```
Comment (slax57) [inline webapp/config/packages/monolog.yaml:7]: Isn't this redundant with `SYMFONY_DEPRECATIONS_HELPER=weak`? :thinking:

File: webapp/public/styles/app.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+.tile::before,
+.tile::after {
+    position: absolute;
+    content: "";
+    background-color: #ffffff;
+    z-index: 0;
+}
+
+.tile-shape-0::before {
```
Comment (slax57) [inline webapp/public/styles/app.css:57]: `.tile-shape-0`, `.tile-shape-1`, `.tile-shape-2`

Why not call them using the letters, just like in the CLI?

**Review by slax57 — CHANGES_REQUESTED** [review]

File: webapp/public/styles/app.css (hunk 3 above)
Comment (jonathan-marmelab) [reply webapp/public/styles/app.css:57]: It avoids me to parse the json and patch it for view

**Comment by slax57** [comment]
It appears you also need the following PHP extensions: `php8.1-xml`, `php8.1-mbstring` and `php8.1-curl`.
You need to document them and/or add a `Makefile` target to install them.

**Comment by slax57** [comment]
Otherwise I've tested the webapp and it works fine, well done! :muscle:

**Review by slax57 — APPROVED** [review]

**Pull request merged**